serverless/packages/functions/src/user/tests/get_user.test.ts

```diff
@@ -50,7 +50,7 @@ it("test no existing user id", async () => {
 });
 
 it("test malformed request", async () => {
-    
+  // @ts-expect-error : we are trying to cause an error
```

Comment:
nit: bit unnecessary to say "we are trying to cause an error" since the test case is self explanatory